src/main.zig

```diff
@@ -1250,6 +1250,7 @@ fn buildOutputType(
                     .framework => try frameworks.append(it.only_arg),
                     .nostdlibinc => want_native_include_dirs = false,
                     .strip => strip = true,
+                    .include_dir => try clang_argv.appendSlice(it.other_args),
```

Comment:
This is identical to `.other`, do you really need `include_dir` ?

Reply:
Thanks @LemonBoy. I think this patch was so old that it became useless. Sorry for the noise. I'm going to close this.